docs/serving/istio-authorization.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "Knative application under the strict authorization policy"
+weight: 25
+type: "docs"
+---
+
+When you deployed app to Knative Serving, serving system pods such as activator and autoscaler access to your app.
+Hence, you have to allow the requests to your app when you configure security features such as istio authorization policy.
+
+> Tip: This example assumes that your application enabled istio sidecar injection.
```

Comment:
This should be a prerequisite instead, and I think this whole doc should be somewhere specific to Istio.
@evankanderson @mattmoor I think we should probably have some sort of "Post-installation tasks" section that includes things like this but I'm not sure, wdyt?